src/android/toga_android/window.py

```diff
@@ -41,7 +43,18 @@ def create_toolbar(self):
         pass
 
     def show(self):
-        pass
+        if not self.widget:
+            print("[Android] Showing an empty window is not supported.")
+            return
+
+        # Set the app's entire contentView to this window: only one Window can
+        # be displayed at a time.
+        self.app.native.setContentView(self.widget.native)
+
+        # Attach child widgets to the this window as their container.
+        for child in self.widget.interface.children:
+            child._impl.container = self.widget
+            child._impl.viewport = self.widget.viewport
```

Comment:
Similarly, this is something I'd expect to see as the consequence of set_content or set_child, rather than show().